eng/pipelines/outerloop.yml

```diff
@@ -1,5 +1,13 @@
 trigger: none
 
+schedules:
+- cron: "0 3 * * *"
+  displayName: Outerloop scheduled build
+  branches:
+    include:
+    - master
+    - releases/3.0
```

Comment:
This should be `release/3.0`.

Anyway I don't think we had outerloop test runs for 3.0 active before, do we need them? This is the setting the build had before:

```json
 "schedules": [
        {
          "branchFilters": [
            "+master"
          ],
          "timeZoneId": "Pacific Standard Time",
          "startHours": 3,
          "startMinutes": 0,
          "daysToBuild": "all",
          "scheduleJobId": "72d01713-e7d9-489d-8313-42486b1895f8",
          "scheduleOnlyWithChanges": true
        }
      ],
```

Also, this probably should be condition to when the build name is corefx-outerloop, cause I would guess  this now adds scheduled builds for, `corefx-outerloop`, `corefx-outerloop-linux, `corefx-outerloop-windows`, `corefx-outerloop-osx` which all share the same `yml` file.

Reply:
thanks, will submit a follow-up PR.